amd_openvx/openvx/include/vx_ext_amd.h

```diff
@@ -203,9 +209,7 @@ enum vx_image_attribute_amd_e
     VX_IMAGE_ATTRIBUTE_AMD_OPENCL_BUFFER = VX_ATTRIBUTE_BASE(VX_ID_AMD, VX_TYPE_IMAGE) + 0x01,
     /*! \brief GPU buffer offset. Use a <tt>\ref cl_uint</tt> parameter.*/
     VX_IMAGE_ATTRIBUTE_AMD_GPU_BUFFER_OFFSET = VX_ATTRIBUTE_BASE(VX_ID_AMD, VX_TYPE_IMAGE) + 0x02,
-    /*! \brief Enable user kernel's own GPU buffer for virtual images. Supports only images with
-     * single color plane and stride should match framework's internal alignment. image ROI not supported.
-     * Use a <tt>\ref vx_bool</tt> parameter.*/
+    /*! \brief Enable user kernel's own GPU buffer for virtual images. Supports only images with single color plane and stride should match framework's internal alignment. image ROI not supported. Use a <tt>\ref vx_bool</tt> parameter.*/
```

Comment:
If it was changed to single line because of warnings it is fine otherwise revert back